tests/unit/FirebaseArray.spec.js

```diff
@@ -570,7 +570,7 @@ describe('$firebaseArray', function () {
 
     it('should apply $$defaults if they exist', function() {
       var arr = stubArray(null, $firebaseArray.$extend({
-        $$defaults: {aString: 'not_applied', foo: 'foo'}
+        $$defaults: {aString: 'alpha', foo: 'foo'}
```

Comment:
Why were these changed? The tests should still be valid as-is if the new code works. Also, "not_applied" describes the desired outcome in this case where "alpha" is non-descript.